application/wso2verify/app/src/androidTest/java/com/wso2/WSO2Verify/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.wso2.test1
```

Comment:
Let's change the package name to follow IS convension

Reply:
Removed this class since it was a dummy test

Reply:
This was removed since it is a dummy test